lib/ClangImporter/ClangIncludePaths.cpp

```diff
@@ -186,7 +186,8 @@ static bool shouldInjectLibcModulemap(const llvm::Triple &triple) {
 static SmallVector<std::pair<std::string, std::string>, 2>
 getLibcFileMapping(ASTContext &ctx, StringRef modulemapFileName,
                    std::optional<StringRef> maybeHeaderFileName,
-                   const llvm::IntrusiveRefCntPtr<llvm::vfs::FileSystem> &vfs) {
+                   const llvm::IntrusiveRefCntPtr<llvm::vfs::FileSystem> &vfs,
+                   std::optional<StringRef> maybeSecondHeaderFileName = {}) {
```

Comment:
Can we change the signature to instead be `std::optional<ArrayRef<StringRef>>` for the headers?